### ISSUE TITLE
Gound Battle

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#279]
```
Game version: 0.94.180

Operating System: (e.g., Windows 7 x86, Windows XP 64-bit)
Windows XP x86 SP3

Java runtime version: (run java -version)
Version6 Update30

Installed using the Launcher? (yes, no)
Yes

Game language (en, hu):
EN

What steps will reproduce the problem?
1. Attacked by AI Enemy - Ground Battle 

What is the expected output? What do you see instead?
1.Problem: Moving or attacking with the own units is not possible.
2.Problem: I have to place all my units with a single left mouse click. It 
should be possible to place them all with rollover+hold right mouseclick.
3.Problem: Its not possible to surrender with shut down the own fortress.
4. Problem: After AI Attack, if you lost the fight the colony is always 
destroyed. 

Please provide any additional information below.
Tested only the Human vs. AI Single Player Modus
```

Original issue reported on code.google.com by `axiral...@gmx.net` on 5 Feb 2012 at 7:32

**Comment by GoogleCodeExporter** [comment]
```
1.) Moving is done via Shift+Right click, attack is done CTRL+Right click in 
modern mode, simple right click in classic mode.
2.) Will be implemented in the next major release
3.) I was not aware of this option, Will be implemented in the next major 
release
4.) If you lose a planet, the population is reduced by 1500-3000 in the current 
version. Repeated attacks will surely destroy them. I'll reduce the losses in 
the next release.
```

Original comment by `akarn...@gmail.com` on 5 Feb 2012 at 7:56
- Changed state: **Started**
- Added labels: **Milestone-0.95**

**Comment by GoogleCodeExporter** [comment]
Original comment by `akarn...@gmail.com` on 5 Feb 2012 at 8:01

**Comment by GoogleCodeExporter** [comment]
```
Problem 3 now implemented in 0.94.181
```

Original comment by `akarn...@gmail.com` on 7 Feb 2012 at 9:50
- Changed state: **SVN**

**Comment by GoogleCodeExporter** [comment]
```
Continuous unit placement will be managed in a separate issue.
```

Original comment by `akarn...@gmail.com` on 10 Feb 2012 at 1:08
- Changed state: **Fixed**
